testing/src/main/java/org/oppia/android/testing/espresso/TextInputAction.kt

```diff
@@ -0,0 +1,46 @@
+package org.oppia.android.testing.espresso
+
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import javax.inject.Inject
+
+/**
+ * Action for matching error text and performing other actions on TextInputLayout in a test infrastructure-specific way.
+ */
+class TextInputAction @Inject constructor() {
```

Comment:
As no we have made our functions static, no need for `@Inject constructor()`. 
So, now we can directly use the functions in our test file like `TextInputAction.hasErrorText(...)` and same for other.

Reply:
Once this is done, check if bazel is building correctly on ci check or not and assign me back.

Reply:
![Screenshot from 2021-07-30 18-05-13](https://user-images.githubusercontent.com/76056229/127653924-171bc145-a00a-420d-b6a3-db150e29a77e.png)
@anandwana001 I did the requested changes. But I am having problem with this as I changed the "View" in the TextInputAction.kt to "TextInputLayout" according to a comment on the previous PR but it still expects a View.

Reply:
I will take a look at it tomorrow morning or later tonight.

Reply:
I am not getting this error when checking on my local. 

<img width="559" alt="Screenshot 2021-08-02 at 22 05 05" src="https://user-images.githubusercontent.com/25057618/127894649-478dcfee-99da-4095-994c-354edae07fd3.png">

Reply:
Did you try running the test?

Reply:
Done.